### PR TITLE
Tsne function & Tests

### PR DESCRIPTION
**Pull request by bosticc**

### CODE REVIEW

**Comment by bosticc** [comment]
should be just three files, deleted all the extra ones

**Comment by zkoogle** [comment]
Please also delete file_path_utils and the test. They are already merged, right?

**Comment by bosticc** [comment]
> Please also delete file_path_utils and the test. They are already merged, right?

yup, sorry bout that.

File: cloud_functions/tsne_embedding.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        Args:
+            trigger_file: 
+                Dictionary that contains data specific to the event.
+            _:
+                Cloud functions event metadata.
+    """
+
+    # Looks for all embeddings in the embedding folder to ensure they are all t
+    # here
+    folder = '/'.join(trigger_file['name'].split('/')[:-1])
```
Comment (zkoogle) [inline cloud_functions/tsne_embedding.py:50]: Can you fix this as well? Here and in all other files in other PRs.

**Comment by bosticc** [comment]
should be just one file now? I think this merge conflict message is old ^^

File: cloud_functions/tsne_embedding.py (hunk 1 above)
Comment (bosticc) [reply cloud_functions/tsne_embedding.py:50]: yup

**Pull request merged**